apps/redis-sentinel/templates/services/app/svc.yml.j2

```diff
@@ -29,6 +40,17 @@ spec:
       fsGroup: {{ redis_sentinel_fsgroup_id }}
   redis:
     replicas: {{ redis_sentinel_redis_replicas }}
+    affinity:
+      podAntiAffinity:
+        preferredDuringSchedulingIgnoredDuringExecution:
+          - weight: 100
+            podAffinityTerm:
+              labelSelector:
+                matchExpressions:
+                  - key: app.kubernetes.io/component
+                    operator: In
+                    values:
+                      - redis
```

Comment:
if I look at what we did on edxapp, you are missing the most important line: "topologyKey: kubernetes.io/hostname"?

Reply:
I don't think we have to use it, this topologykey is specific to the selector we use on the other DC (https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#built-in-node-labels) here it's a custom label added by the operator. I don't know if there is a topologykey for this case.

Reply:
You are right and I am wrong. I added the topologykey